Makefile

```diff
@@ -307,6 +307,16 @@ clean-mobile:
 	$(RM) -r mobile/build
 	$(RM) mobile/*_generated.go
 
+docs:
+	@$(call print, "Generating man pages documents.")
+	if [ -f ./lnd-debug ] && [ -f ./lncli-debug ]; then \
+	./lnd-debug --manpage; \
```

Comment:
Can we just depend on the `build` step to make sure the `lnd-debug` and `lncli-debug` files are present?